conda/environments/cugraph_dev_cuda10.1.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - cudf=0.18.*
 - libcudf=0.18.*
 - rmm=0.18.*
-- cuxfilter=0.17.*
+- cuxfilter=0.18.*
```

Comment:
You should update https://github.com/rapidsai/cugraph/blob/branch-0.18/ci/release/update-version.sh so this automated

Reply:
Thanks, I didn't even notice that script.  I just updated, tested, and pushed it.